galaxy/main/celerytasks/user_notifications.py

```diff
@@ -113,7 +117,41 @@ def email_verification(email, code, username):
 
 
 @celery.task
-def import_status(task_id, user_initiated, has_failed=False):
+def collection_import(task_id):
```

Comment:
Not getting notifications for failed collection imports.

Reply:
this was only triggering sometimes since the task sometimes was still in the 'running' state, fixed by passing import status to notification